Jenkinsfile

```diff
@@ -12,10 +12,6 @@ pipeline {
   }
 
   parameters {
-    booleanParam(name: 'skipGitHubSite',
-      description: 'If checked the plugin documentation on GitHub will NOT be updated',
-      defaultValue: true)
-

```

Comment:
I wouldn't include these github site changes in this PR...  it mixes up things that don't relate.
And I think today I won't work to backport it just like this. There's more work to do which may is not easily done. I've yet only implemented the "modern" deployment for 12
https://github.com/axonivy/project-build-plugin/pull/534/files
... we may backport it, but then better in an own change.

Reply:
Does the old way still work? Because then I can also trigger it manually here. I think there will not be much more progress on this version...

Reply:
I didn't work last time due to missing egit artifacts. But I assume that we could get it fixed with minimal efforts by doing what Alex did last time to solve it: #528 ... to me it would be perfectly fine to just add this fix for 10. And not invest any further to keep this old thing alive.